ludwig/features/h3_feature.py

```diff
@@ -117,8 +118,12 @@ def call(self, inputs, training=None, mask=None):
     def get_input_dtype(cls):
         return tf.uint8
 
-    def get_input_shape(self):
-        return H3_VECTOR_LENGTH,
+    # def get_input_shape(self):
+    #     return H3_VECTOR_LENGTH,
+
+    @property
+    def input_shape(self) -> torch.Size:
+        return torch.Size((H3_VECTOR_LENGTH, ))
```

Comment:
For consistency, I would use a list instead of a tuple here.

Reply:
Done